merge/multiple_appliers_test.go

```diff
@@ -985,7 +985,7 @@ func (r repeatingConverter) Convert(v *typed.TypedValue, version fieldpath.APIVe
 	if err != nil {
 		return nil, missingVersionError
 	}
-	y, err := yaml.Marshal(v.AsValue().Unstructured())
+	y, err := yaml.Marshal(v.AsValue().AsUnstructured())
```

Comment:
That's not really a case either, that's slightly different, I'd keep it as `Unstructured`